package.json

```diff
@@ -33,6 +33,7 @@
     "@ecl/twig-component-inpage-navigation": "3.0.2",
     "@ecl/twig-component-language-list": "3.0.2",
     "@ecl/twig-component-link": "3.0.2",
+    "@ecl/twig-component-label": "3.0.2",
```

Comment:
Since we added the label component here, I think we should update the oe-theme-helper-call-label-status.html.twig to use it instead of hardcoding it.